src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -64,49 +64,57 @@ public TestType asTestCall(@NotNull PsiElement element) {
     return null;
   }
 
-  Map<DartCallExpression, TestType> cachedCallToTestType;
+  private final Map<String, Map<Integer, TestType>> cachedCallToTestType = new HashMap<>();
+  private final Map<String, FlutterOutline> pathToPreviouslyUsedOutline = new HashMap<>();
```

Comment:
nit: rather than two maps with the same key, define a class  with both the 
FltuterOutline and the Map<Integer, TestType> as fields.

Reply:
Done.